src/RegistrationController.js

```diff
@@ -52,42 +61,61 @@ function (Okta, BaseLoginController, FormType, Enums, BaseSchema, SchemaFormFact
   return BaseLoginController.extend({
     className: 'registration',
     initialize: function() {
-      var Schema = BaseSchema.Model.extend({
-        url: 'api/v1/schema',
-        expand: ['schema']
+      var Schema = RegistrationSchema.Schema.extend({
+        url: 'api/v1/schema'
       });
       var schema = new Schema({
         schema: {
-          'properties':{
-            'firstname': {
+          'properties': {
+            'firstName': {
               'type': 'string',
-              'description': 'First name',
+              'description': 'First Name',
+              'default': 'Enter your first name',
               'maxLength': 255
             },
-            'lastname': {
+            'lastName': {
               'type': 'string',
-              'description': 'Last name',
+              'description': 'Last Name',
+              'default': 'Enter your last name',
               'maxLength': 255
             },
-            'email': {
-              'type': 'email',
-              'description': 'Email'
+            'login': {
+              'type': 'string',
+              'description': 'Email Address',
+              'format' : 'email',
+              'default': 'Enter your email',
+              'maxLength': 255
+            },
+            'accountLevel': {
+              'type': 'string',
+              'description': 'Account Level',
+              'enum': [ 'Free', 'Premium', 'Platinum' ]
             },
-            'password': {
-              'type': 'password',
-              'description': 'Password'
+            'referrer': {
+              'type': 'string',
+              'description': 'How did you hear about us?',
+              'maxLength': 1024
+            },
+            'password' : {
+              'type' : 'string',
+              'description' : 'Password'
```

Comment:
We use the description for the placeholder on the input ? Should it be called placeholder?